packages/buendia-dashboard/data/usr/share/buendia/dashboard/downtime

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+# Copyright 2015 The Project Buendia Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License.  You may obtain a copy
+# of the License at: http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distrib-
+# uted under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, either express or implied.  See the License for
+# specific language governing permissions and limitations under the License.
+
+up_count=$(cat /var/log/buendia/buendia-warmup.log | grep '(end, status 0)' | wc -l)
+down_count=$(cat /var/log/buendia/buendia-warmup.log | grep '(end, status 1)' | wc -l)
+downtimes=$(cat /var/log/buendia/buendia-warmup.log | grep '(end, status 1)' | sed -e 's/:[0-9][0-9]\W*[0-9]*> .*/<br\/>/')
+
+# I would just use bc but it's not installed on Debian so we're constrained to integer arithmetic
```

Comment:
You can install `bc` if you want. If you don't want to, a better comment would be "It's not worth installing `bc` just for this calculation so we constrain this calculation to integer arithmetic".